### ISSUE TITLE
fail to delete a watson discovery instance

### ISSUE THREAD

**Issue by huikang** [IBM-Bluemix/bluemix-go#56]
Hi, I saw the following error when deleting a discovery instance. However, I could not find any field in the bluemix client config. Could anyone help? Thanks.

```json
{
  "description": "The following error occurred in the region 'US South': Service instance test-discovery: Service broker error: {\"error\"=>\"AsyncRequired\", \"description\"=>\"This service plan requires client support for asynchronous service operations.\"}",
  "error_code": "CF-ServiceBrokerBadResponse",
  "code": 10001,
  "http": {
    "uri": "https://provision-broker.ng.bluemix.net/bmx/provisioning/brokers/1867973154-7bc1c9a9-9fd7-462f-930e-96f5557c2f5a/v2/service_instances/bdfd19ba-f8ad-4a97-979a-aff3e3a59329",
    "method": "DELETE",
    "status": 422
  },
```

**Comment by huikang** [comment]
Here is the link to the API: https://apidocs.cloudfoundry.org/272/service_instances/delete_a_service_instance.html

**Comment by ashishth09** [comment]
This sounds more like API enhancement.

**Comment by huikang** [comment]
@ashishth09 I can look into this, if you don't mind.

**Comment by huikang** [comment]
Fixed by PR #57

**Comment by huikang** [comment]
Thanks, @ashishth09
